docker-compose.yml

```diff
@@ -1,17 +1,17 @@
-# version: "3.7"
+version: "3.7"
 
-# services:
-#   web:
-#     build: .
-#     environment:
-#       NODE_ENV:
-#       MONGODB_URI: mongodb://mongo:27017/nodegoat
-#     command: sh -c "until nc -z -w 2 mongo 27017 && echo 'mongo is ready for connections' && node artifacts/db-reset.js && npm start; do sleep 2; done"
-#     ports:
-#       - "4000:4000"
+services:
+  web:
+    build: .
```

Comment:
#### Automated PR Comment From coverity
<!-- 7bc3bf1fe4113d6abbbb2403ed9c19b3 -->
<!-- Comment managed by Git Commenter, do not modify! -->
### Coverity Issue - Container requests ability to craft raw network packets
_High CWE-269_
The Docker container requests the `NET_RAW` capability, either explicitly or by default, granting access to the host's network interfaces.
### How to fix
Explicitly remove the `NET_RAW` capability by adding either `NET_RAW` or `ALL` to the `cap_drop` list and avoiding the `NET_RAW` capability in the `cap_add` list.